### DISCUSSION TITLE
Next steps for onu-ui

### DISCUSSION THREAD

**Discussion by yzh990918** [onu-ui/onu-ui#64] [General]
## 🧤🧤Component Feature list(WIP)

**Here's a list of components I'd like to write**

Layout, Divider and Grid components in Layout component; icon component with embedded code copy and search icon features; Text layout component (Typography); Carousel component; Collapse panel component (reconstruction); Comment component; List component; Description list component; Image component; Value jump component; Table component; Tabs component; Timeline component; Tree component; Result page; Input; Digital input; Form; Checkbox; Radio; Tag input; Cascading selector; Selector; Slider; Switch; Textarea; Time selector; Calendar; Data shuttle box; Tree selector; Upload; Step bar; Tooltip; Drawer; Modal; Notification; Pop confirm; Progress bar; Loading; Skeleton screen; Breadcrumb; Dropdown; Menu; Header; Pagination component; Back to top; Anchor.

**🦄🦄 Support dynamic unplugin theme config**.

**Comment by wzc520pyfm** [comment]
Our components are in dire need of development, and I think community contributors can claim the components to be developed here first, and then develop them after they have been approved by the management. This can avoid conflict and duplication of work.

**Comment by Tauromachian** [comment]
What kind of grid do you intend to use? If its something "classical" like Bootstrap's grid I have experience and can get you started
